src/Tests/Helpers/ServiceBusMessageHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace SFA.DAS.Funding.SystemAcceptanceTests.Helpers
+{
+    internal class ServiceBusMessageHelper
+    {
+        private readonly ScenarioContext _context;
+        private ApprenticeshipCreatedEvent _apprenticeshipCreatedEvent;
+        private EarningsGeneratedEvent _earnings;
+        private FundingPeriod _funding;
+
+        public ServiceBusMessageHelper(ScenarioContext context)
+        {
+            _context = context;
+        }
+
+        public void CreateAnApprenticeshipMessage(DateTime actualStartDate, DateTime plannedEndDate, decimal agreedPrice)
+        {
+            var fixture = new Fixture();
+            _apprenticeshipCreatedEvent = fixture.Build<ApprenticeshipCreatedEvent>()
```

Comment:
I don't think this is the right event, we need to be using the one from approvals.